.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2.3.1
+        with:
+         python-version: "3.6"
```

Comment:
Can we use Python 3.8?

Reply:
Yeah we can , actually the azure pipeline had 3.6 version thats why i took this